packages/core/src/Breadcrumb.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Icon from 'pcln-icons'
+import Text from './Text'
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Breadcrumb({ className, isCurrent, isCompleted, children, onClick }) {
+  const color = isCurrent || isCompleted ? 'primary' : 'text.light'
+
+  return (
+    <Container className={className} alignItems="center" onClick={onClick}>
+      {isCompleted && <CheckIcon />}
+      <Text color={color} bold={isCurrent} fontSize={1}>
+        {children}
+      </Text>
+    </Container>
+  )
+}
+
+const CheckIcon = () => (
+  <Icon name="check" color="primary" size={16} mr={1} legacy={false} />
+)
+
+Breadcrumb.displayName = 'Breadcrumb'
+
+Breadcrumb.propTypes = {
+  className: PropTypes.string,
```

Comment:
Can you explain this className prop? I wouldn’t imagine CSS-in-JS would use class name.

Reply:
When using styled component or any CSS-in-JS library the component has to accept a className otherwise the styles will not take effect.